front/app/api/phases/types.ts

```diff
@@ -159,8 +159,7 @@ export type IdeaDefaultSortMethod =
   | 'random'
   | 'popular'
   | 'new'
-  | '-new'
-  | null;
+  | '-new';
 
```

Comment:
Not sure why, but on the BE we allow nil values for the ideas_order, so I think we should probably leave in "null" in the type for now? :

![image](https://github.com/user-attachments/assets/9d1f5001-45e8-4bb1-8fc8-50291d6b7829)

Reply:
@amanda-anderson Sending `ideas_order` with the request is not required (as your line confirms), so `undefined` is fine. Also, in our FE, `defaultParticipationConfig` gets overwritten by specific participation method configs and the value is never `undefined` in practice. (`defaultParticipationConfig` is exported once, but it shouldn't even be there. Added a comment and I'll add it to our clean-up ticket.)